.travis.yml

```diff
@@ -1,11 +1,12 @@
 language: scala
 scala:
-  - 2.11.7
+  - 2.11.12
+  - 2.12.6
 jdk:
-  - oraclejdk8
+  - openjdk8
 
 script:
-  - sbt clean coverage test coverageReport && sbt coverageAggregate
+  - sbt clean coverage +test coverageReport && sbt coverageAggregate
```

Comment:
Why are you doing `coverageAggregate` in a separate SBT? Might be worth a comment at least.